app/models/induction_period.rb

```diff
@@ -12,10 +12,15 @@ class InductionPeriod < ApplicationRecord
             presence: true
 
   validates :number_of_terms,
-            inclusion: { in: 0..16,
-                         message: "Terms must be between 0 and 16" },
-            presence: { message: "Enter a number of terms" },
-            if: -> { finished_on.present? }
+            inclusion: {
+              in: 0..16, message: "Terms must be between 0 and 16", if: -> { finished_on.present? }
+            },
+            presence: {
+              message: "Enter a number of terms", if: -> { finished_on.present? }
+            },
+            absence: {
+              message: "Delete the number of terms if the induction has no end date", if: -> { finished_on.blank? }
+            }
```

Comment:
Should this be in `PendingInductionSubmission` too? Maybe they can be together in `SharedInductionPeriodValidation` (although the optional validation context might restrict that)